playbooks/cicognara.yml

```diff
@@ -1,15 +1,11 @@
 ---
 # by default this playbook runs in the staging environment
 # to run in production, pass '-e runtime_env=production'
-- name: build the cicognara site
```

Comment:
This playbook can be deleted entirely. We do not want to build a standalone cicognara site any more.